server/data/formClient.js

```diff
@@ -7,7 +7,7 @@ const sequenceClause =
 module.exports = {
   getFormDataForUser(bookingId) {
     const query = {
-      text: `select id, booking_id, user_id, status, form_response, assigned_user_id, referred_date, referred_by, security_reviewed_date, security_reviewed_by
+      text: `select id, booking_id as "bookingId", user_id, status, form_response, assigned_user_id, referred_date, referred_by, security_reviewed_date, security_reviewed_by
```

Comment:
I think that removing the database column names from the code would be an improvement. If you agree - I'll change the other ones